controllers/grid/users/chapter/form/ChapterForm.inc.php

```diff
@@ -90,17 +90,23 @@ function initData() {
 
 		$monograph = $this->getMonograph();
 		$this->setData('submissionId', $monograph->getId());
+		$chapterPublicationDatesEnabledSubmission = (int)$this->getMonograph()->getOnlineFirst() === 1;
```

Comment:
Shouldn't this function name change as well (`getOnlineFirst`)?

Reply:
thanks @asmecher , changed.